libnmstate/nm/checkpoint.py

```diff
@@ -124,6 +126,21 @@ def create(self):
                 raise NMCheckPointPermissionError(str(e))
             raise NMCheckPointCreationError(str(e))
 
+        GLib.timeout_add(
+            self._timeout * 0.5, self._refresh_checkpoint_timeout, None,
+        )
+
+    def _refresh_checkpoint_timeout(self, data):
+        cancellable, cb, cb_data = (None, None, None)
+        nm_client = client()
+        if nm_client:
+            nm_client.checkpoint_adjust_rollback_timeout(
+                self._dbuspath, self._timeout, cancellable, cb, cb_data
+            )
+            return True
+        else:
```

Comment:
Please add a comment here explaining why NM.Client might be None here.

Reply:
I've adapted your comment about this. Thanks